src/containers/Network/index.js

```diff
@@ -66,7 +64,11 @@ const Network = ({
           location.pathname === '/network/client-devices' ? (
             <div className={styles.headerContent}>
               <ToggleButton activeTab={activeTab} />
-              <Button onClick={onReload} title="reload" icon={<ReloadOutlined />} />
+              <Button
+                onClick={() => onReload(location.pathname)}
+                title="reload"
+                icon={<ReloadOutlined />}
+              />
```

Comment:
redo it this is not proper React flow, move the toggle and refresh buttons to the APs and Clients page.